openmdao/docs/features/building_components/explicitcomp.rst

```diff
@@ -16,6 +16,13 @@ ExplicitComponent Methods
 
 The implementation of each method will be illustrated using a simple explicit component that computes the output *area* as a function of inputs *length* and *width*.
 
+::
```

Comment:
that does import the code, but not in a box in the code style like the rest of the code.. may be possible to style it?